contracts/nois-proxy/src/contract.rs

```diff
@@ -164,6 +169,7 @@ fn execute_get_next_randomness(
     job_id: String,
 ) -> Result<Response, ContractError> {
     let config = CONFIG.load(deps.storage)?;
+    let callback_address = info.clone().sender;
```

Comment:
No need to clone the whole info, you can just clone the sender:

```suggestion
    let callback_address = info.sender.clone();
```